test/ibmcloud.status.test.js

```diff
@@ -303,25 +360,283 @@ describe('Test cloud status via Reg Ex', function() {
 			this.timeout(5000);
 			testMonitoringServiceStatus('Sydney', 'www', 'UP', [{
 				color: UP_COLOR,
-				text: 'www in Sydney with UP status.',
-				title: 'Service monitoring started'
+				text: i18n.__('monitoring.status', 'www', 'Sydney', 'UP'),
+				title: i18n.__('monitoring.started')
 			}], [{
 				color: UP_COLOR,
-				text: 'Service is UP.',
-				title: 'www in Sydney Region',
+				text: i18n.__('service.status', 'UP'),
+				title: i18n.__('service.in.region', 'www', 'Sydney'),
 				title_link: 'http://estado.au-syd.bluemix.net'
 			}], true, done);
 		});
 	});
 
+	context('Monitoring service with any/clear', function() {
+
+		let testMonitoringServiceStatusAny = function(region, service, expectedFirstReplyAttachments, expectedSecondReplyAttachments, expectedThirdReplyAttachments, expectedFourthReplyAttachments, statusChange, done) {
+			let regionCode = REGIONS[region];
+			if (statusChange) {
+				// First status
+				nock('http://estado.' + regionCode + '.bluemix.net')
+					.get('/')
+					.reply(200, mockHtml[regionCode]);
+				// Second status (same)
+				nock('http://estado.' + regionCode + '.bluemix.net')
+					.get('/')
+					.reply(200, mockHtml[regionCode]);
+				// Third status (updated)
+				nock('http://estado.' + regionCode + '.bluemix.net')
+					.get('/')
+					.reply(200, mockHtml[regionCode + '-updated']);
+				// Fourth status (back to original)
+				nock('http://estado.' + regionCode + '.bluemix.net')
+					.get('/')
+					.reply(200, mockHtml[regionCode]);
+			}
+			else {
+				// Same status four times
+				nock('http://estado.' + regionCode + '.bluemix.net')
+					.get('/')
+					.times(4)
+					.reply(200, mockHtml[regionCode]);
+			}
+			let count = 0;
+			room.robot.on('ibmcloud.formatter', function(event) {
+				count++;
+				try {
+					if (count === 1) {
+						expect(event.attachments).to.deep.equal(expectedFirstReplyAttachments);
+					}
+					else if (count === 2) {
+						expect(event.attachments).to.deep.equal(expectedSecondReplyAttachments);
+						if (!statusChange) {
+							done();
+						}
+					}
+					else if (count === 3) {
+						if (statusChange) {
+							expect(event.attachments).to.deep.equal(expectedThirdReplyAttachments);
+						}
+						else {
+							done(new Error('Unexpected third attachments: ' + event.attachments));
+						}
+					}
+					else if (count === 4) {
+						if (statusChange) {
+							expect(event.attachments).to.deep.equal(expectedFourthReplyAttachments);
+							done();
+						}
+						else {
+							done(new Error('Unexpected fourth attachments: ' + event.attachments));
+						}
+					}
+				}
+				catch (err) {
+					done(err);
+				}
+			});
+			room.user.say('user1', '@hubot ibmcloud status monitor ' + region + ' any ' + service).then(setTimeout(function() {
```

Comment:
No.  Unfortunately, it is not arbitrary.  The test interval for checking for status changes is 1000ms.  These particular tests require 4 intervals to occur (one to get the initial status, the next to have no status change, the next to see a down->up status change, and the last to see an up->down status change).  So 4500ms gives enough time for these events to occur with a little buffer.

It does take extra time to run the tests, but I kept the number to as few as I could.